Scripts/Interfaces/Capability/IFeedbackReciever.cs

```diff
@@ -0,0 +1,14 @@
+﻿#if UNITY_EDITOR
+namespace UnityEditor.Experimental.EditorVR
+{
+	/// <summary>
+	/// Implementors can receive Feedback Requests
+	/// </summary>
+	public interface IFeedbackReciever
+	{
+		void AddFeedbackRequest(FeedbackRequest request);
+		void RemoveFeedbackRequest(FeedbackRequest request);
+		void ClearFeedbackRequests(IRequestFeedback caller);
```

Comment:
But the function clears all feedback requests if no caller is supplied.

Reply:
Hm. This is an interesting one. It's actually set up so that you can't call it without a caller bc the extension method always sends along the caller. The case where we call it with null can only happen from the module.

Thanks for bringing this up, because all three of these methods should probably have docs, even though it's an internal interface (for now).